src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.util.TaskList;
+
+/**
+ * Represents a command that can be executed.
+ */
+public abstract class Command {
+    /**
+     * Returns false if the command is not an ExitCommand.
+     */
+    public boolean isExit() {
+        return false;
+    }
```

Comment:
Good use on this 👍 ! I did something similar but naively made it an abstract method which meant that I had to implement it in all my subclasses 😓